src/Oasis/Signer.cpp

```
@@ -0,0 +1,197 @@
+#define TRANSFER_METHOD "staking.Transfer"
+#define ESCROW_METHOD "staking.AddEscrow"
+#define RECLAIM_ESCROW_METHOD "staking.ReclaimEscrow"
+
+using namespace TW;
+
+namespace TW::Oasis {
+
+Proto::SigningOutput Signer::sign(const Proto::SigningInput& input) noexcept {
+    auto output = Proto::SigningOutput();
+    auto signer = Signer(input);
+    auto encoded = signer.build();
```

cppcheck (2.10-2)

[error] Exception thrown in function declared not to throw exceptions.

(throwInNoexceptFunction)